src/sparseml/exporters/transforms/matmul_add_to_matmulinteger_add_cast_mul.py

```diff
@@ -41,7 +42,7 @@ class MatMulToMatMulIntegerAddCastMul(OnnxTransform):
     |         |
     | input   Dq
     |   |     |
-    |  Q/Dq   Transpose
+    |  Q/Dq   optional Transpose
```

Comment:
do we test when this exists and doesn't exist? Potentially recall some issues around optional when doing intiial opt experiments

Reply:
Yeah it wasn't supported when specified in parents, part of this PR fixes that bug.

There's a test added also in this PR for it